web/app/js/components/ConfigureProfilesMsg.jsx

```diff
@@ -140,7 +140,9 @@ class ConfigureProfilesMsg extends React.Component {
   render() {
     const { showAsIcon } = this.props;
 
-    if (showAsIcon) {
+    if (this.props.singleNamespace === "true") {
```

Comment:
😭

Reply:
Heh, yeah, I cried a bit too when I wrote this...